### ISSUE TITLE
Add training set accuracy check

### ISSUE THREAD

**Issue by lpf123321** [ChawDoe/LeNet5-MNIST-PyTorch#10]
Would it possible to print the **accuracy of training set** in each epoch?

By comparing the two accuracy of training set and testing set, it is convenient to check whether there is an overfitting.
